content/en/docs/ambient/usage/l7-features/index.md

```diff
@@ -128,3 +127,40 @@ spec:
       port: 9080
       weight: 10
 {{< /text >}}
+
+## Unsupported APIs with waypoints
+
+Some legacy Istio APIs are deliberately not supported by waypoints in ambient mode.  These APIs can still be used with [Istio Gateways](/docs/tasks/traffic-management/ingress/ingress-control/).
+
+### VirtualService
+
+Istio's legacy traffic routing API is not supported for configuring waypoint traffic routing, though it works in some circumstances.
+
+Any use of VirtualService with waypoints is considered Alpha, and may be subject to change in future releases.
+Istio's maintainers do not intend to remove this support, but will not be progressing it to [any further feature phase](/docs/releases/feature-stages).
+
+#### Migrating from VirtualService to Gateway API routes
+
+[Only a single VirtualService](/docs/reference/config/analysis/ist0109/) can be used for mesh traffic matching a certain hostname. However, multiple Gateway API routes can refer to the same host.
+
+This is especially relevant when you are migrating from VirtualService to Gateway API routes. If you create one or more HTTPRoutes which specify a Service that is also in use with a VirtualService, the HTTPRoute/s will apply and the VirtualService will not.
+
+To avoid this situation, users migrating from sidecars should look to convert their VirtualService configuration to Gateway API routes. The [ingress2gateway](https://github.com/kubernetes-sigs/ingress2gateway/) project has limited support for this use case. Caution is advised, especially for users who use [subset routing](/docs/concepts/traffic-management/#destination-rules)
+
+#### Using features that are not in Gateway API
+
+A small number of Istio's features cannot currently be expressed in Gateway API: for example, [fault injection](/docs/tasks/traffic-management/fault-injection/) and [direct response](/docs/reference/config/networking/virtual-service/#HTTPDirectResponse). It is technically possible to use VirtualService for these use cases, as long as the configured `hosts` do not conflict with the `parentRefs` of any Gateway API route as mentioned above.
+
+#### DestinationRule subsets
+
+Gateway API has no ability to address [subsets](/docs/reference/config/networking/destination-rule/#Subset). Instead, you must define additional Services which have a more granular selector than the original.
+
+The other features of DestinationRule are supported.
```

Comment:
exportTo is also not supported, but I think @Stevenjin8's PR covers that?

Reply:
Is `workloadSelector` supported? 
If not we could basically say "only `trafficPolicy` is supported"

Reply:
https://github.com/istio/istio/pull/51085/ isn't clear, so I'm guessing it does work